models/pathology_nuclei_segmentation_classification/configs/train.json

```diff
@@ -0,0 +1,502 @@
+{
+    "imports": [
+        "$import glob",
+        "$import os",
+        "$import skimage"
+    ],
+    "bundle_root": "$os.getcwd()",
+    "ckpt_dir_stage0": "$@bundle_root + '/models/stage0'",
+    "ckpt_dir_stage1": "$@bundle_root + '/models/stage1'",
+    "ckpt_path_stage0": "$@ckpt_dir_stage0 + '/checkpoint_epoch=50.pt'",
```

Comment:
why such hardcoding?
why can't we follow the similar standard by referring: 
https://github.com/Project-MONAI/model-zoo/blob/dev/models/spleen_ct_segmentation/configs/train.json
https://github.com/Project-MONAI/model-zoo/blob/dev/models/spleen_ct_segmentation/configs/train.json#L255


```
    "ckpt_dir": "$@bundle_root + '/models'",
    "output_dir": "$@bundle_root + '/eval'",
```

as per the bundle spec.. you try to produce model.pt (which can serve the current best model as part of your training pipeline)...

i strongly suggest to read through the existing (spleen) example and fix some of these things..

Reply:
@SachidanandAlle Maybe that is fine considering that is a two stage training unlike the other bundles. 
I personally think that it is better to have one standard bundle is a way that the user be able to run it with different input arguments for two-stage training. What do you think @KumoLiu?

Reply:
i am ok with staging.. my main point is `checkpoint_epoch=50.pt`

Reply:
Oh yeah, agree. That is already fixed.